packages/desktop/hooks/LocalCombatsContext/index.tsx

```diff
@@ -58,13 +60,19 @@ export const LocalCombatsContextProvider = (props: IProps) => {
           console.log(
             `${wowVersion} ShuffleRoundEnded Round ${combat.sequenceNumber}, killed: ${combat.roundEndInfo.killedUnitId}`,
           );
+          setCombats((prev) => {
+            return prev.concat([combat]);
+          });
         }
       });
 
       window.wowarenalogs.logs?.handleSoloShuffleEnded((_event, combat) => {
         if (wowVersion === combat.wowVersion) {
           console.log('ShuffleEnded');
           console.log(combat);
+          setCombats((prev) => {
```

Comment:
i wonder if we want to store shuffle matches in a separate array. each shuffle round is very similar to an arena match can we can likely handle them in a consistent way. but shuffle matches need an entirely different UX and upload behavior. thoughts?